docs/fides/docs/development/quickstart.md

```diff
@@ -0,0 +1,25 @@
+# Quickstart
+
+This guide will walk you through building a local development environment to start contributing quickly! Be sure to review our [community guidelines](https://docs.ethyca.com/fides/community/overview) and happy hacking!
```

Comment:
This feels redundant with `Developing Fides`, I think it would be a better dev experience to have all of this info in one place?

What was your reasoning for wanting to put this on a separate page?